clients/src/main/java/org/apache/kafka/common/requests/LeaderAndIsrRequest.java

```diff
@@ -42,26 +42,52 @@
 
 public class LeaderAndIsrRequest extends AbstractControlRequest {
 
+    public enum Type {
+        Unknown(0),
```

Comment:
enum names should be capitalized (surprised checkstyle didn't complain about this)